tests/appsec/test_blocking_addresses.py

```diff
@@ -27,6 +27,7 @@ class Test_BlockingAddresses:
     def setup_request_method(self):
         self.rm_req = weblog.request("OPTIONS")
 
+    @released(ruby="?")
```

Comment:
Ultimately that one should work, we just miss the address for Rack, right?

IIRC there's the case where a verb can be turned into behaving like another with some Rails parameter though (something like having a parameter named `_method` or something) for back when AJAX could only handle `GET` and `POST` but one wanted RESTful `PUT`, `DELETE`, `PATCH`. Not sure if it's still possible in recent Rails given that browsers now all support these cases proper.

Also not sure WAF+rules are prepared to support that.